app/lib/actions/katello/content_view_version/auto_create_products.rb

```diff
@@ -0,0 +1,22 @@
+module Actions
+  module Katello
+    module ContentViewVersion
+      class AutoCreateProducts < Actions::Base
+        def plan(organization:, metadata:)
+          helper = ::Katello::Pulp3::ContentViewVersion::ImportableProducts.
```

Comment:
I think this class should have some kind of verb in the name like Helper/Manager/Doer to be consistent with the rest of our service/helper type classes

Reply:
I did call them helpers before and then backed out because I thought the title was clear enough. I am ok renaming it if you prefer Helper/Manager

Reply:
I do prefer it for consistency but I wouldn't NACK this PR for it :)